src/tribol/mesh/MfemData.cpp

```diff
@@ -295,7 +295,7 @@ void MfemMeshData::SetParentCoords( const mfem::ParGridFunction& current_coords
   coords_.SetParentGridFn( current_coords );
 }
 
-void MfemMeshData::UpdateMfemMeshData()
+void MfemMeshData::UpdateMfemMeshData( RealT binning_proximity_scale )
```

Comment:
Not a big deal, but it looks a little funny to have this routine take the binning proximity scale. Seems like only the coupling scheme needs to know about the binning proximity scale. Is it because the binning scale needs to be modified by the lor factor, and that is internal to redecomp, not at the coupling scheme level, and therefore, the mfem mesh here needs to know about it?

Reply:
This is for creating the redecomp mesh, which doesn't know anything about a coupling scheme.